src/gui/qgsfilewidget.h

```diff
@@ -120,6 +121,17 @@ class GUI_EXPORT QgsFileWidget : public QWidget
      */
     void setFilter( const QString &filter );
 
+    /**
+     * Sets whether a confirmation to overwrite an existing file will appear
+     * \param confirmOverwrite If set to true, an overwrite confirmation will be shown
```

Comment:
Should the default value of this property be documented?

Reply:
@m-kuhn , done.

Reply:
Thanks.

FYI, Personally I started to document such setter/getter pairs as properties and just copy paste the content. Without an extra description for the setter parameter which actually only states the obvious.

Makes them much easier to write and focus on what the *property* does and not what the *setter* does (except of course the setter has some very special magic in it).

```
Defines whether... Defaults to...
```